src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorHelpers.cs

```diff
@@ -0,0 +1,624 @@
+			// on the other side of the spectrum.  Otherwise, we'll place it on the boundary that was exceeded.
+			if (originalAlpha < minBound)
+			{
+				originalAlpha = shouldWrap ? maxBound : minBound;
+			}
+
+			if (originalAlpha > maxBound)
+			{
+				originalAlpha = shouldWrap ? minBound : maxBound;
+			}
+
+			// We multiplied alpha by 100 previously, so now we want to put it back in the 0-1 range.
+			return originalAlpha / 100;
+		}
+
+		public static async void CreateCheckeredBackgroundAsync(
+			int width,
+			int height,
+			Color checkerColor,
+			List<byte> bgraCheckeredPixelData,
+			IAsyncAction asyncActionToAssign,
+			CoreDispatcher dispatcherHelper,
+			Action<WriteableBitmap> completedFunction)
+		{
+			if (width == 0 || height == 0)
+			{
+				return;
+			}
+
+			bgraCheckeredPixelData.Capacity = width * height * 4;
+
+			//WorkItemHandler workItemHandler =
+			//(IAsyncAction workItem) =>
+			await Task.Run(() =>
+			{
+				for (int y = 0; y < height; y++)
+				{
+					for (int x = 0; x < width; x++)
+					{
+						//if (workItem.Status == AsyncStatus.Canceled)
+						//{
+						//	break;
+						//}
+
+						// We want the checkered pattern to alternate both vertically and horizontally.
+						// In order to achieve that, we'll toggle visibility of the current pixel on or off
+						// depending on both its x- and its y-position.  If x == CheckerSize, we'll turn visibility off,
+						// but then if y == CheckerSize, we'll turn it back on.
+						// The below is a shorthand for the above intent.
+						bool pixelShouldBeBlank = ((x / CheckerSize) + (y / CheckerSize)) % 2 == 0 ? true : false;
+
+						if (pixelShouldBeBlank)
+						{
+							bgraCheckeredPixelData.Add(0);
+							bgraCheckeredPixelData.Add(0);
+							bgraCheckeredPixelData.Add(0);
+							bgraCheckeredPixelData.Add(0);
+						}
+						else
+						{
+							bgraCheckeredPixelData.Add((byte)(checkerColor.B * checkerColor.A / 255));
+							bgraCheckeredPixelData.Add((byte)(checkerColor.G * checkerColor.A / 255));
+							bgraCheckeredPixelData.Add((byte)(checkerColor.R * checkerColor.A / 255));
+							bgraCheckeredPixelData.Add(checkerColor.A);
+						}
+					}
+				}
+			});
+
+			//if (asyncActionToAssign != null)
+			//{
+			//	asyncActionToAssign.Cancel();
+			//}
+
+			//asyncActionToAssign = ThreadPool.RunAsync(workItemHandler);
+			//asyncActionToAssign.Completed = new AsyncActionCompletedHandler(
+			//async (IAsyncAction asyncInfo, AsyncStatus asyncStatus) =>
+			//{
+			//	if (asyncStatus != AsyncStatus.Completed)
+			//	{
+			//		return;
+			//	}
+
+			//	asyncActionToAssign = null;
+
+				// Uno Doc: Assumed normal priority is acceptable
+				await dispatcherHelper.RunAsync(CoreDispatcherPriority.Normal, () =>
+				{
+					WriteableBitmap checkeredBackgroundBitmap = CreateBitmapFromPixelData(width, height, bgraCheckeredPixelData);
+					completedFunction?.Invoke(checkeredBackgroundBitmap);
+				});
+			//});
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)